packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_table_row.dart

```diff
@@ -25,18 +29,45 @@ class LoggingTableRow extends StatefulWidget {
 
   final bool isSelected;
 
-  static TextStyle get metadataStyle =>
-      Theme.of(navigatorKey.currentContext!).subtleTextStyle;
+  static TextStyle get metadataStyle {
+    final currentContext = navigatorKey.currentContext;
+    assert(
+      currentContext != null,
+      'LoggingTableRow.metadataStyle requires a valid navigatorKey to be set. If this assertion is hit in tests then make sure to `wrap()` the widget being pumped.',
+    );
+    return Theme.of(navigatorKey.currentContext!).subtleTextStyle;
+  }
 
   static TextStyle get detailsStyle =>
       Theme.of(navigatorKey.currentContext!).regularTextStyle;
 
+  /// All of the metatadata chits that can be visible for this [data] entry.
+  @visibleForTesting
+  static List<MetadataChit> metadataChits(LogDataV2 data, double maxWidth) {
```

Comment:
nit: can we rename every use of Chit to Chip? Fits material design terminology better :) thanks!